### ISSUE TITLE
JMP accepts input from xbox gamepad

### ISSUE THREAD

**Issue by thisisnotmyrealname** [jellyfin/jellyfin-media-player#325]
2022-10-25 15:54:16 [ INFO  ] JS: Received KeepAlive from server.
2022-10-25 15:54:36 [ DEBUG ] InputComponent.cpp @ 160 - Input received: source: "Xbox One S Controller" keycode: "KEY_BUTTON_4" : InputBase::KeyDown
2022-10-25 15:54:36 [ DEBUG ] InputComponent.cpp @ 230 - Emit input action: ("seek_backward")
2022-10-25 15:54:36 [ INFO  ] JS: Uncaught TypeError: Cannot read property 'rewind' of null
2022-10-25 15:54:36 [ DEBUG ] InputComponent.cpp @ 160 - Input received: source: "Xbox One S Controller" keycode: "KEY_BUTTON_4" : InputBase::KeyUp
2022-10-25 15:54:40 [ DEBUG ] InputSDL.cpp @ 153 - JoyAxisMotion: 3 191
2022-10-25 15:54:40 [ DEBUG ] InputSDL.cpp @ 153 - JoyAxisMotion: 2 2179
2022-10-25 15:54:40 [ DEBUG ] InputSDL.cpp @ 153 - JoyAxisMotion: 2 2118

**Describe the bug**
Jellyfin Media Player on a fresh install of Ubuntu 22.04.01 64bit will accept input from an Xbox One S Controller even if the "Enable Gamepad" option is UNCHECKED.

**To Reproduce**

Install Jellyfin Media Player
Pair Xbox One S Controller
Play media in Jellyfin
Try to play a game in Steam

**Expected behavior**
Jellyfin Media Player should ignore input from a gamepad if the "Enable Gamepad" option is unchecked

**Desktop (please complete the following information):**
 - OS: Kubuntu 22.04.01 5.15.0-52-generic
 - Version 1.7.1-1
 - Installation Method jammy deb file

**Additional context**
Originally had the flatpak installed, happened there as well. uninstalled flatpak, used the deb, issue persists.

**Comment by iwalton3** [comment]
The `Enable Gamepad` option is provided by the web client and only controls gamepad mapping in web browsers. (I believe QT WebEngine also supports some degree of controller binding too, which further complicates these issues.)

I believe you can also disable this by editing the `sdlEnabled` option in the `jellyfinmediaplayer.conf` file.

**Comment by gnometsunami** [comment]
I am also seeing this bug with the linux native app. And I can confirm that editing the conf file successfully disables gamepad input.

If you installed jellyfin using flatpak, the path to the conf file is `~/.var/app/com.github.iwalton3.jellyfin-media-player/data/jellyfinmediaplayer/jellyfinmediaplayer.conf`

**Comment by m-GDEV** [comment]
In case you used a different installation method: https://github.com/jellyfin/jellyfin-media-player#config-file-location

**Comment by sevenrats** [comment]
any objections to closing this resolved issue?

**Comment by JeremiahFaust** [comment]
> I am also seeing this bug with the linux native app. And I can confirm that editing the conf file successfully disables gamepad input.
> 
> If you installed jellyfin using flatpak, the path to the conf file is `~/.var/app/com.github.iwalton3.jellyfin-media-player/data/jellyfinmediaplayer/jellyfinmediaplayer.conf`

For the Arch Linux user's like myself who installed the AUR package, the jellyfinmediaplayer.conf can be found here instead:
/.local/share/jellyfinmediaplayer/jellyfinmediaplayer.conf

**Comment by seeseemelk** [comment]
> any objections to closing this resolved issue?

Yes, as it does not seem resolved to me. Modifying the inputmap seems more like a workaround, as I too expected that turning of "Enable gamepad" would disable my gamepad.

It seems to me that this could only be considered resolved once either that option clearly documents the fact that it does not do what one would expect, or make it behave it a way that users expect.

**Comment by Mondrethos** [comment]
> > any objections to closing this resolved issue?
> 
> Yes, as it does not seem resolved to me. Modifying the inputmap seems more like a workaround, as I too expected that turning of "Enable gamepad" would disable my gamepad.
> 
> It seems to me that this could only be considered resolved once either that option clearly documents the fact that it does not do what one would expect, or make it behave it a way that users expect.

I agree with this. Having "Enable Gamepad" disabled, should disable gamepads, yet I can't casually run my jellyfin desktop client in one monitor and game in the other using a controller because the client still accepts the input globally.

**Comment by andrewrabert** [comment]
This Qt desktop client has been deprecated in favor of a rewrite: https://github.com/jellyfin/jellyfin-desktop
